blob/fileblob/fileblob.go

```diff
@@ -565,6 +582,20 @@ func (b *bucket) Delete(ctx context.Context, key string) error {
 }
 
 func (b *bucket) SignedURL(ctx context.Context, key string, opts *driver.SignedURLOptions) (string, error) {
-	// TODO(Issue #546): Implemented SignedURL for fileblob.
+	path, err := b.path(key)
+	if err != nil {
+		return "", err
+	}
+
+	return b.urlSigner.SignedURL(ctx, key, path, opts)
```

Comment:
Why does SignedURL need `path`? Isn't `key` sufficient?

Reply:
I chatted about this with @zombiezen, as I had the same question,  and his comment (please correct me if I'm paraphrasing this inaccurately) was that it will need the actual path info to serve the file, but since the dir information to assemble the full path is only available directly on the bucket, one cannot derive the full path from the key OR the key from the full path.